<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QDialog" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#333333;</string>
  </property>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>323</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>323</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_password_prompt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>123</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Password Prompt on startup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:white;</string>
   </property>
   <property name="text">
    <string>Change RaSk0  Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_new_pass">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>40</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_confrim_new">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>77</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_show_pass">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>109</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_con_pas_mess">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pass_message">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_min_idle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>145</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_idle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Idle Time until Password Pormpt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoExit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>176</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Auto-Exit Timer</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_autoExit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>173</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>178</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_please_wait">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>326</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_portable">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>201</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_export">
      <property name="styleSheet">
       <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
      </property>
      <property name="text">
       <string>Export Program and Data to USB drive</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_backup">
      <property name="styleSheet">
       <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
      </property>
      <property name="text">
       <string>Backup Data File </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_restorbackup">
      <property name="styleSheet">
       <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
      </property>
      <property name="text">
       <string>Retore Data File </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
